src/sentry/static/sentry/app/views/organizationHealth/styles/healthPanelChart.jsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'react-emotion';
+
+import PanelChart from 'app/components/charts/panelChart';
+import space from 'app/styles/space';
+
+const getChartMargin = () => `
+  margin-right: ${space(2)};
+  &:last-child {
+    margin-right: 0;
+  }
+`;
+
+const HealthPanelChart = styled(PanelChart)`
+  ${getChartMargin};
```

Comment:
curious if there's any special reason this is a function?

Reply:
This gets re-used in another branch, but we should change it from a function to  `css` since it never takes any args.